pkg/kube/client.go

```diff
@@ -63,6 +64,16 @@ func New(getter genericclioptions.RESTClientGetter) *Client {
 
 var nopLogger = func(_ string, _ ...interface{}) {}
 
+// Test connectivity to the Client
+func (c *Client) IsReachable() {
+	client, _ := c.Factory.KubernetesClientSet()
+	_, err := client.ServerVersion()
+	if err != nil {
+		c.Log("--- Kubernetes Cluster Unreachable ---")
+		os.Exit(1)
```

Comment:
Instead of performing an `os.Exit` can you return an error. Then where this code is used can you check for an error and return it up the stack.